libscript/src/script-execute.cpp

```diff
@@ -116,7 +116,7 @@ class MCScriptForeignInvocation
 	{
 		if (m_argument_count == kMaxArguments ||
 			!Allocate(sizeof(void *),
-					  __alignof__(void *),
```

Comment:
@livecodefraser Do you think it might be better to put a `#define __alignof__ sizeof` in `foundation.h` (protected by appropriate platform-specific `#ifdef`ery)?  I think this would make it easier to fix properly later on.